rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/TclMavenLauncher.xtend

```diff
@@ -47,12 +47,12 @@ public class TclMavenLauncher implements TclLauncher {
 		// val testCases = createTestCasesCommaList(selection)
 		val result = mavenExecutor.executeInNewJvm(parameters, project.location.toOSString,
 			"test=" + testCases.join(","), monitor)
-		val testResultFile = project.createOrGetDeepFolder(MVN_TEST_RESULT_FOLDER).location.toFile
+		val testResultFolder = project.createOrGetDeepFolder(MVN_TEST_RESULT_FOLDER).location.toFile
 		if (result != 0) {
-			logger.
-				error('''Error during maven build using parameters='«parameters»' and element='«testCases»'.''')
+			logger.error('''Error during maven build using parameters='«parameters»' and element='«testCases»'.''')
```

Comment:
I think we should use in all classes the same logging strategy logger.error("some message about {} and {}.", "Foo","Bar")